site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -163,6 +163,8 @@ the `message GameServer`.
 For language specific documentation, have a look at the respective source (linked above), 
 and the {{< ghlink href="examples" >}}examples{{< /ghlink >}}.
 
+You can use `GameServerCaching` feature passed as a feature gate if you want a `GameServer` to be returned right after `GameServerFeature` is called.
```

Comment:
Can we also wrap this is a feature shortcode, so it only shows up in the next release please :+1: 

```suggestion
You can use `GameServerCaching` feature passed as a feature gate if you want a `GameServer` to be returned right after `SDK.WatchGameServer(...)` is called.
```

Might also want to link to  https://agones.dev/site/docs/guides/feature-stages/#feature-gates here as well for information on how to turn this on.